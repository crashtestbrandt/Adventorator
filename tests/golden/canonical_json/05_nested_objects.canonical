{"outer_a":{"inner_m":4,"inner_z":3},"outer_z":{"inner_a":2,"inner_b":1}}