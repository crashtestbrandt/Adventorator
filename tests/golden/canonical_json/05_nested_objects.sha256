bbc2456a44eb68a56e0bfb08280d5b4f702362b3d0e4e61f5554e650fecc62f6
