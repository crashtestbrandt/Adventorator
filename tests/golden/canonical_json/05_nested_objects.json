{
  "outer_z": {
    "inner_b": 1,
    "inner_a": 2
  },
  "outer_a": {
    "inner_z": 3,
    "inner_m": 4
  }
}